ocs_ci/ocs/resources/pod.py

```diff
@@ -1077,6 +1077,63 @@ def get_topolvm_node_pod(
     return Pod(**topolvm_node_pod[0])
 
 
+def get_noobaa_core_pod(
+    label: str = constants.NOOBAA_CORE_POD_LABEL,
+    namespace: str = config.ENV_DATA["cluster_namespace"],
+) -> Pod:
+    """
+    Get noobaa core pod
+
+    Args:
+        label (str): Label associated with noobaa-core pod
+        namespace (str): Namespace in which noobaa-core pod is residing
+
+    Returns:
+        Pod: Pod object of noobaa-core pod
+
+    """
+    noobaa_core_pod = get_pods_having_label(label, namespace)
+    return Pod(**noobaa_core_pod[0])
+
+
+def get_noobaa_endpoint_pod(
+    label: str = constants.NOOBAA_ENDPOINT_POD_LABEL,
+    namespace: str = config.ENV_DATA["cluster_namespace"],
+) -> Pod:
+    """
+    Get Noobaa endpoint pod
+
+    Args:
+        label (str): Label associated with noobaa endpoint pod
+        namespace (str): Namespace in which noobaa endpoint pod is residing
+
+    Returns:
+        Pod: Pod object of noobaa endpoint pod
+
+    """
+    noobaa_endpoint_pod = get_pods_having_label(label, namespace)
+    return Pod(**noobaa_endpoint_pod[0])
+
+
+def get_noobaa_operator_pod(
+    label: str = constants.NOOBAA_OPERATOR_POD_LABEL,
+    namespace: str = config.ENV_DATA["cluster_namespace"],
+) -> Pod:
+    """
+    get Noobaa operator pod
+
+    Args:
+        label (str): Label associated with noobaa operator pod
```

Comment:
could you please rebase? these methods implementations are already there

Reply:
done